server.js

```diff
@@ -7,8 +7,24 @@
 
 // list the food trucks returned by filterByDay, passing in the current day name.
 // e.g. filterByDay(day); where "day" is determined using the enhanced-date module
-// The list of trucks returned will be an array of food truck objects. Iterate 
-// through the list, building up a string of food truck names. Once you're done 
+// The list of trucks returned will be an array of food truck objects. Iterate
+// through the list, building up a string of food truck names. Once you're done
 // iterating through that list, display the string you built up.
 
-// Remember that the response is a stream object that must be closed.
\ No newline at end of file
+// Remember that the response is a stream object that must be closed.
+
+var http = require('http');
+var dateObj = require('./enhanced-date');
+var trucks = require('./trucks');
+var truckList = trucks.filterTrucksByDay(dateObj.getDayName());
+var date = new Date(dateObj.getDate({format : 'milliseconds'})).getDate();
```

Comment:
Getting the truck list and the date are better done inside the request handler. A server can run for days, but this will always return the same values as when the server first started.